sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UserTest.java

```diff
@@ -113,6 +116,15 @@
     };
   }
 
+  @DataProvider(name = "getUserByLoginIdFailure")
+  public Object[][] getUserByLoginIdFailure() {
+    return new Object[][] {
+      new Object[] {"testGetUserByLoginIdFailureWithInvalidLoginId"},
+      new Object[] {"testGetUserByLoginIdFailureWithEmptyLoginId"},
+      new Object[] {"testGetUserByLoginIdFailureWithInvalidRequest"}
```

Comment:
Rename testGetUserByLoginIdFailureWithInvalidRequest to testGetUserByLoginIdFailureWithNullLoginId.